spec/features/webauthn/management_spec.rb

```diff
@@ -540,7 +540,7 @@ def expect_webauthn_platform_setup_error
       mock_press_button_on_hardware_key_on_setup
 
       expect(current_path).to eq webauthn_setup_path
-      expect(page).to have_content t('errors.webauthn_platform_setup.unique_name')
+      expect(page).to have_content t('errors.webauthn_setup.unique_name')
```

Comment:
Suspect this was a bug disguised by the previous. sameness between webauthn_setup.unique and webauthn_platform_setup.unique.
A nickname doesn't get added in the webauthn_platform setup workflow.